IdentityServer/Quickstart/Account/AccountController.cs

```diff
@@ -91,6 +92,22 @@ public async Task<IActionResult> Login(string returnUrl)
                                         });
             }
 
+            // get the provider parameter from the return url
```

Comment:
```suggestion
            // get the provider parameter from the return url
            if(vm.ReturnUrl == null)
            {
                vm.ReturnUrl = "";
            }
```
Does not break existing functionality with the frontend, this does fix an issue where you can't navigate to /account/login anymore.